tests/LOOP/test_35292.py

```diff
@@ -19,51 +19,74 @@
 class test_main(GaiaTestCase):
 
     def setUp(self):
-        #
-        # Set up child objects...
-        #
         GaiaTestCase.setUp(self)
         self.UTILS = UTILS(self)
         self.contacts = Contacts(self)
         self.loop = Loop(self)
-        #
-        # Get details of our test contacts.
-        #
-        self.contact = MockContact()
-        self.UTILS.general.insertContact(self.contact)
+
+        self.test_contact = MockContact()
+        curr_time = repr(time.time()).replace('.', '')
+        self.test_contact['email'] = [self.test_contact['email']]
+        self.test_contact['tel'] = [self.test_contact['tel']]
+        self.test_contact['email'].append({
+            'type': 'Job',
+            'value': '{}@testmail.net'.format(curr_time)})
+        self.test_contact['tel'].append({
+            'type': 'Landline',
+            'value': '91{}'.format(curr_time)})
+        self.UTILS.general.insertContact(self.test_contact)
         self.data_layer.connect_to_wifi()
 
-        result = self.loop.initial_test_checks()
+        self.loop.initial_test_checks()
 
+        result = self.loop.wizard_or_login()
         if result:
             self.loop.phone_login_auto()
             self.loop.allow_permission_phone_login()
             self.UTILS.element.waitForElements(DOM.Loop.app_header, "Loop main view")
-
         self.apps.kill_all()
         time.sleep(2)
+
         _ = setup_translations(self)
-        self.expected_message = _("No problem! Just share the following link and they can call you back from"\
-                                  " any browser.")
+        msg = "{} is not a Firefox Hello user yet. No problem! Just create a room and share it with him."
```

Comment:
This string is spread all over the tests, maybe it could be put somewhere in the TK